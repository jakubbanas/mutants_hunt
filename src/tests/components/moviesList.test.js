import React from "react";
import { shallow } from "enzyme";

import MoviesList from "../../components/moviesList";

it("renders without crashing", () => {
  const wrapper = shallow(<MoviesList />);
  expect(wrapper).toMatchSnapshot();
});
