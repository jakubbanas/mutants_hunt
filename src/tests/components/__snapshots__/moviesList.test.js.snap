// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MoviesList />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Movie
          image="https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg"
          rating={4}
          title="Spider man"
        />,
        <Movie
          image="https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg"
          rating={1}
          title="Spider man"
        />,
        <Movie
          image="https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg"
          rating={0}
          title="Spider man"
        />,
        <Movie
          image="https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg"
          rating={2}
          title="Spider man"
        />,
        <Movie
          image="https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg"
          rating={0}
          title="Spider man"
        />,
        <Movie
          image="https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg"
          rating={5}
          title="Spider man"
        />,
        <Movie
          image="https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg"
          rating={0}
          title="Spider man"
        />,
      ],
      "className": "row",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": "0",
        "nodeType": "class",
        "props": Object {
          "image": "https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg",
          "rating": 4,
          "title": "Spider man",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "1",
        "nodeType": "class",
        "props": Object {
          "image": "https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg",
          "rating": 1,
          "title": "Spider man",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "2",
        "nodeType": "class",
        "props": Object {
          "image": "https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg",
          "rating": 0,
          "title": "Spider man",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "3",
        "nodeType": "class",
        "props": Object {
          "image": "https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg",
          "rating": 2,
          "title": "Spider man",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "4",
        "nodeType": "class",
        "props": Object {
          "image": "https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg",
          "rating": 0,
          "title": "Spider man",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "5",
        "nodeType": "class",
        "props": Object {
          "image": "https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg",
          "rating": 5,
          "title": "Spider man",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "6",
        "nodeType": "class",
        "props": Object {
          "image": "https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg",
          "rating": 0,
          "title": "Spider man",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Movie
            image="https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg"
            rating={4}
            title="Spider man"
          />,
          <Movie
            image="https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg"
            rating={1}
            title="Spider man"
          />,
          <Movie
            image="https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg"
            rating={0}
            title="Spider man"
          />,
          <Movie
            image="https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg"
            rating={2}
            title="Spider man"
          />,
          <Movie
            image="https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg"
            rating={0}
            title="Spider man"
          />,
          <Movie
            image="https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg"
            rating={5}
            title="Spider man"
          />,
          <Movie
            image="https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg"
            rating={0}
            title="Spider man"
          />,
        ],
        "className": "row",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "0",
          "nodeType": "class",
          "props": Object {
            "image": "https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg",
            "rating": 4,
            "title": "Spider man",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "1",
          "nodeType": "class",
          "props": Object {
            "image": "https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg",
            "rating": 1,
            "title": "Spider man",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "2",
          "nodeType": "class",
          "props": Object {
            "image": "https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg",
            "rating": 0,
            "title": "Spider man",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "3",
          "nodeType": "class",
          "props": Object {
            "image": "https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg",
            "rating": 2,
            "title": "Spider man",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "4",
          "nodeType": "class",
          "props": Object {
            "image": "https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg",
            "rating": 0,
            "title": "Spider man",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "5",
          "nodeType": "class",
          "props": Object {
            "image": "https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg",
            "rating": 5,
            "title": "Spider man",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "6",
          "nodeType": "class",
          "props": Object {
            "image": "https://www.gameaxis.com/wp-content/uploads/2018/09/spider-man-hero.jpg",
            "rating": 0,
            "title": "Spider man",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
